socs/agents/starcam_lat/agent.py

```diff
@@ -0,0 +1,277 @@
+        try:
+            self.StarcamHelper = StarcamHelper(ip_address, port)
+        except socket.timeout:
+            self.log.error("Starcamera connection has times out")
+            return False, "Timeout"
+
+    @ocs_agent.param('_')
+    def send_commands(self, session, params=None):
+        """
+        send_commands()
+
+        **Process**
+        packs and sends camera+astrometry-related commands to starcam
+
+        **Return**
+        returns a touple with True/False and a string describing whether
+        or not a lock could be acquired and commands were sent to the sc
+        """
+        with self.lock.acquire_timeout(job='send_commands') as acquired:
+            if not acquired:
+                self.log.warn(f"Could not start Task because "
+                              f"{self._lock.job} is already running")
+                return False, "Could not acquire lock"
+            self.log.info("Sending commands")
+            self.StarcamHelper.pack_and_send_cmds()
+        return True, "Sent commands to starcamera"
+
+    @ocs_agent.param('_')
+    def acq(self, session, params=None):
+        """
+        acq()
+
+        **Process**
+        acquires data from starcam and publishes to feed
+
+        **Return**
+<<<<<<< HEAD
+        once the acq() loop exits (wherein data is retrieved from
+        the camera and pulished), a touple with True/False and a string
+        describing whether or not the loop was exited after the end of
+        an acquisition.
+=======
+        once the acq() loop exits (wherein data is retrieved from the camera and pulished),
+        a touple with True/False and a string describing whether or not the loop was exited
+        after the end of an acquisition.
+>>>>>>> 82252e753327de73c687591486c64dca8f4a8638
+        """
+        if params is None:
+            params = {}
+        with self.lock.acquire_timeout(timeout=100, job='acq') as acquired:
+            if not acquired:
+                self.log.warn("Could not start init because {} is already
+                              running".format(self.lock.job))
+                return False, "Could not acquire lock"
+            session.set_status('running')
+            self.log.info("Starting acquisition")
+            self.take_data = True
+            while self.take_data:
+                data = {
+                    'timestamp': time.time(),
+                    'block_name': 'astrometry',
+                    'data': {}
+                }
+                # get astrometry data
+                astrom_data = self.StarcamHelper.get_astrom_data()
+<<<<<<< HEAD
+                # update the data dictionary+session and publish
+                data['data'].update(astrom_data_dict)
+=======
+                # update the data dictionary, update the session, and publish
+                data['data'].update(astrom_data_dict)
+>>>>>>> 82252e753327de73c687591486c64dca8f4a8638
```

Comment:
This is indicative of a merge that wasn't handled properly. When there's a merge conflict the conflicting blocks are presented side by side like this and must be merged manually. Please resolve the three places this happened in this commit: https://github.com/simonsobs/socs/pull/335/commits/04999b45f227904e29f233b05f70265ea79f52e2

Reply:
will resolve and push again - thanks for the checkup